block/stream/output/state_changes.proto

```diff
@@ -810,4 +854,4 @@ message QueuePushChange {
  * information beyond the state name common to all state changes.
  */
 message QueuePopChange {
-}
+}
```

Comment:
A newline is required at EOF (isn't enforced yet due to minimal CI support).

Reply:
Ah yes, added.